README.md

```diff
@@ -1,16 +1,52 @@
-# Kroma
+<div align="center">
+  <br />
+  <br />
+  <a href="https://kroma.network"><img alt="Kroma" src="https://github.com/kroma-network/kroma-brand-kit/raw/main/assets/images/signature/Kroma-signature.svg" width=500></a>
+  <br />
+  <h3><a href="https://kroma.network">Kroma</a> aims to be a New Universal ZK Rollup on Ethereum.</h3>
+  <br />
+</div>
 
-This is a client for Kroma network.
+## About Kroma
 
-## Documentation
+Kroma aims to develop a universal ZK Rollup based on the Optimism Bedrock architecture. Currently, Kroma operates as an
+Optimistic Rollup with ZK fault proofs, utilizing a zkEVM based on Scroll. The goal of Kroma is to eventually transition
+to a ZK Rollup once the generation of ZK proofs becomes more cost-efficient and faster.
 
-If you want to build Kroma, check out the [Protocol Specs](./specs/).
+Kroma offers 10x lower transactions fees compared to the Ethereum mainnet, with native Ethereum security and Ethereum
+equivalence. Our roadmap to transition from Optimistic rollup to ZK rollup sets Kroma apart from other L2 networks.
+Additionally, everyone in Kroma has the opportunity to participate as a validator. You can read our
+article '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
+to learn more.
+
+Kroma is live on the mainnet on 09/05/23, and we're working on optimizing ZK Proofs to transition to ZK Rollup as part
+of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.
```

Comment:
I think this is not necessary here. But let's add the word `the first universal state 2 rollup` at the start of this description.